README.asciidoc

```diff
@@ -958,7 +958,7 @@ Some options are built in Kakoune, and can be used to control its behaviour:
    - `option=<opt-name>` where <opt-name> is a _completions_ option.
  * `static_words` _str-list_: list of words that are always added to completion
      candidates when completing words in insert mode.
- * `completions_extra_word_chars` _str_: a string containing all additional character
```

Comment:
Actually, this is not a `str` anymore, its a `codepoint-list` and is `:` separated as other lists.